### PR TITLE
pause-upgrade stability test

### PR DESCRIPTION
**Pull request by shuijing198799**
### What problem does this PR solve? 
      Modified the process in order to properly execute stability testing。
### What is changed and how it works?
      Modify stability test DockerFile and change the location to clean webhoook.yaml
### Check List 

Tests 

 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

### Does this PR introduce a user-facing change?:
 
 ```release-note

 ```
NONE

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -376,6 +376,15 @@ func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 		return fmt.Errorf("failed to deploy operator: %v, %s", err, string(res))
 	}
 
+	// delete statefulset update webhook and configuration
+	cmd = fmt.Sprintf("kubectl delete -f %s/webhook.yaml", oa.manifestPath(info.Tag))
```
Comment (weekface) [inline tests/actions.go]: why move the cleaner to `DeployOperator`?

Comment (shuijing198799) [reply tests/actions.go]: manifests dir will only exist after checkouttag in stability test

**Pull request merged**